test/go.mod

```diff
@@ -20,28 +20,27 @@ require (
 	github.com/integr8ly/integreatly-operator v1.19.1-0.20230418125645-5aec6c015174
 	github.com/integr8ly/keycloak-client v0.1.11
 	github.com/noobaa/noobaa-operator/v5 v5.11.0
-	github.com/onsi/ginkgo/v2 v2.4.0
-	github.com/onsi/gomega v1.23.0
+	github.com/onsi/ginkgo/v2 v2.9.1
+	github.com/onsi/gomega v1.27.3
 	github.com/openshift-online/ocm-sdk-go v0.1.344
 	github.com/openshift/api v3.9.1-0.20191031084152-11eee842dafd+incompatible
-	github.com/operator-framework/api v0.17.3
-	github.com/operator-framework/operator-registry v1.26.2
+	github.com/operator-framework/api v0.17.5
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.58.0
-	github.com/prometheus/client_golang v1.13.0
+	github.com/prometheus/client_golang v1.14.0
 	github.com/redhat-developer/observability-operator/v4 v4.2.1
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sirupsen/logrus v1.9.0
 	golang.org/x/net v0.8.0
 	golang.org/x/term v0.6.0
 	golang.org/x/text v0.8.0
 	google.golang.org/api v0.114.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.25.0
-	k8s.io/apiextensions-apiserver v0.25.0
-	k8s.io/apimachinery v0.25.0
+	k8s.io/api v0.26.3
+	k8s.io/apiextensions-apiserver v0.26.2
```

Comment:
Should we opt for using the same version of the k8s.io package here as the others (v0.26.3)?

Reply:
Good catch, yes I think we should. I'll update 👍